D4Companion.Services/ScreenProcessHandler.cs

```diff
@@ -130,867 +113,514 @@ private void initImageList()
             _imageListItemAspectLocations.Clear();
             _imageListItemAspects.Clear();
 
-            string systemPreset = _settingsManager.Settings.SelectedSystemPreset;
-            string directory = $"Images\\{systemPreset}\\";
-            if (!Directory.Exists(directory))
+            (_thresholdMin, _thresholdMax) = (new Gray(_settingsManager.Settings.ThresholdMin), new Gray(_settingsManager.Settings.ThresholdMax));
+            var baseDirectory = new DirectoryInfo(_settingsManager.Settings.SelectedSystemPresetDir);
```

Comment:
Related to comment on `SettingsD4.cs`. Replace with `SelectedSystemPreset` and prepend the `image` dir yourself.
This change currently breaks the system preset selecting